corehq/apps/app_manager/tests/data/suite/search_command_detail.xml

```diff
@@ -5,7 +5,7 @@
         <locale id="m0.case_short.title"/>
       </text>
     </title>
-    <field sort="default">
+    <field>
```

Comment:
@czue So this is a little odd...
Your addition of `self.app.build_spec = BuildSpec(version='tests', build_number=1)` causes a different test, `test_case_search_action` to fail, unless this change is also made. I haven't looked into why the existence of the `BuildSpec` would make a difference though. Does this change look acceptable? My reading of the spec is that the `sort` attribute of `field`s is obsolete anyways: https://github.com/dimagi/commcare-core/wiki/Suite20